assets/components/svgs/amberZuck.jsx

```diff
@@ -4,8 +4,8 @@ import React from 'react';
 import { gridUrl } from 'helpers/theGrid';
 
 export default function AmberZuck() {
-  const amberUrl = gridUrl('amberRudd', 500, 'png');
-  const zuckUrl = gridUrl('zuck', 500, 'png');
+  const amberUrl = gridUrl('amberRudd', 140, 'png');
+  const zuckUrl = gridUrl('zuck', 140, 'png');
```

Comment:
Does this still look ok? The image will be upscaled I think (140px -> 186px) but maybe not enough to degrade it too much?

Reply:
I think you're right, I've switched to using the larger image